### PR TITLE
Added exceptions for advices and thr to .typos.toml

### PR DESCRIPTION
**Pull request by svetasmirnova**
- [x] The contributed code is licensed under GPL v2.0
- [x] Contributor Licence Agreement (CLA) is signed
- [ ] util/update-modules has been ran
- [ ] Documentation updated
- [ ] Test suite update

### CODE REVIEW

File: .typos.toml [hunk 1]
```diff
@@ -33,6 +33,7 @@ extend-ignore-re = [
 ]
 
 [default.extend-identifiers]
+"advices" = "advices" # typo in version_check.go
```
Comment (artemgavrilov) [inline .typos.toml]: Why not to fix typo instead? Or we have there singular and plural `advise`?

**Review by artemgavrilov — APPROVED** [review]

File: .typos.toml (hunk 1 above)
Comment (svetasmirnova) [reply .typos.toml]: This is the variable name: array of multiple advice. But can change it to something like advs.

Comment (artemgavrilov) [reply .typos.toml]: How about this? https://github.com/percona/percona-toolkit/pull/801

Comment (svetasmirnova) [reply .typos.toml]: Looks good.

**Review by EvgeniyPatlan — APPROVED** [review]

File: .typos.toml (hunk 1 above)
Comment (artemgavrilov) [reply .typos.toml]: Merged

**Pull request merged**